js/index.js

```diff
@@ -26,6 +26,26 @@ window.onload = function(){
     widgets[w].setTextDOM(document.getElementById("widget-" + w + "-text"));
   }
 
+  //load image destinations
+  $.getJSON(config.redditUrl,function(data){
+    try{
+      data["data"]["children"].forEach(element => {
+        if (element["data"]["url"].includes(".jpg")){
+          config.backgroundImages.push(element["data"]["url"])
+        }
+      });
+
+    }catch(err){
+      config.backgroundImages=  [
+        "/img/bunnies.jpg", // https://i.imgur.com/mE1MUpd.jpeg
+        "https://i.imgur.com/nR8jJEM.jpeg",
+        "https://live.staticflickr.com/5443/9369949185_0846c27d6c_b.jpg",
+        "https://live.staticflickr.com/3804/11623216156_9e9f5eaa4a_c.jpg",
+    ]
```

Comment:
Can we set this to pull the defaults from the config file?

Reply:
Fixed this.